solutions/0100-0199/0125-valid-palindrome.md

```diff
@@ -0,0 +1,51 @@
+---
+description: 'Author: @vale-c | https://leetcode.com/problems/valid-palindrome/'
+---
+
+# 0125 - Valid Palindrome (Easy)
+
+## Problem Link
+
+https://leetcode.com/problems/valid-palindrome/
+
+## Problem Statement
+
+A phrase is a palindrome if, after converting all uppercase letters into lowercase letters and removing all non-alphanumeric characters, it reads the same forward and backward. Alphanumeric characters include letters and numbers.
+
+Given a string `s`, return `true` if it is a palindrome, or `false` otherwise.
+
+**Constraints:**
+
+* `1 <= s.length <= 2 * 105`
+* `s` consists only of printable ASCII characters.
+
+
+## Approach: Two Pointers
```

Comment:
add time complexity & space complexity of your solution

Reply:
Good point! I have just added both time/space complexity for this algorithm ✨